template-compose/detekt-config.yml

```
@@ -202,8 +203,8 @@ naming:
     packagePattern: '^[a-z]+(\.[a-z][a-z0-9]*)*$'
   TopLevelPropertyNaming:
     active: true
-    constantPattern: '[A-Z][_A-Z0-9]*'
-    propertyPattern: '[a-z][A-Za-z\d]*'
+    constantPattern: '[A-Z][_A-Za-z0-9]*'
```

@Wadeewee Why don't we use this instead?
```suggestion
    constantPattern: '[A-Z][A-Za-z0-9]*'
```
Ref: https://detekt.dev/docs/introduction/compose/#toplevelpropertynaming

---

@lydiasama We need support in case It doesn't compose `constantPattern`. For example in `OkHttpClientModule.kt`:

```
private const val READ_TIME_OUT = 30L
```

That's why we need to use `[A-Z][_A-Za-z0-9]*` instead of `[A-Z][A-Za-z0-9]*`.

---

@Wadeewee Hmm, it's kind of weird. Since we used the recommended regex in DeeMoney Migration, and it works for all classes. 🤔  @Tuubz @ryan-conway @kaungkhantsoe Do you have any idea?

---

Oh, I was wrong. DeeMoney used this pattern too. 👍🏻